caom2-search-server/src/main/resources/META-INF/resources/cadcVOTV/javascript/cadc.plugin.footprint-viewer.js

```diff
@@ -318,6 +324,27 @@
       }
     }
 
+    function _calculateFootprintsFOV(_footprints) {
+      var maxRA = []
+      var minRA = []
+      var maxDec = []
+      var minDec = []
+      for (var i = 0; i < _footprints.length; i++) {
+        var footprint = _footprints[i]
```

Comment:
For efficient looping, set the length of the array to a variable so as not to keep asking it for the length:
```
for (var i = 0, fl = _footprints.length; i < fl; i++) {
...
```